.github/dependabot.yml

```
@@ -0,0 +1,37 @@
+# See Dependabot documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "docker"
+    directory: "/.ci"
+    schedule:
+      interval: "weekly"
+    reviewers:
+      - "stuartmorgan"
+      - "ditman"
+    labels:
+      - "ecosystem"
+      - "waiting for tree to go green"
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    reviewers:
+      - "hixie"
+      - "godofredoc"
+    labels:
+      - "team"
+      - "ecosystem"
+      - "team: infra"
+      - "waiting for tree to go green"
+  - package-ecosystem: "gradle" 
+    directory: "/" 
+    schedule:
+      interval: "daily"
+    reviewers:
+      - "stuartmorgan"
+      - "ditman"
```

```suggestion
      - "ditman"
      - "blasten"
```

I don't know much of Android, so if some Gradle change comes in, I might not be the best. I suggest adding @blasten (or whoever he feels is most appropriate) to this list.